### PR TITLE
[ROSA-HCP]: add 4.15, 4.16 jobs and remove 4.13

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -941,42 +941,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-aws-sts-shared-vpc
-- as: aws-rosa-sts-hcp-full-f360
```
Comment (yasun1) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:944]: Any reason to remove the 4.16 jobs?

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:944]: Added 4.16 jobs again

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml (hunk 1 above)
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:944]: Shall we update the title to reflect the real change ?

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:944]: Sure, done

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 2]
```diff
@@ -917,66 +917,105 @@ tests:
...
+    workflow: rosa-aws-sts-hypershift-advanced
+- as: aws-rosa-sts-hcp-advanced-stage-full-f360
+  cron: 22 15 18 4 *
   steps:
     cluster_profile: aws-sd-qe
     env:
       CHANNEL_GROUP: nightly
       E2E_RUN_TAGS: '@amd64 and @aws-ipi and @rosa and @hypershift-hosted'
-      OPENSHIFT_VERSION: "4.16"
+      OPENSHIFT_VERSION: "4.14"
```
[the remark this answers is not in the document]
Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]: Sorry, it's just a typo, updated

**Comment by liangxia** [comment]
tide — Not mergeable. PR has a merge conflict.

**Comment by liangxia** [comment]
BTW, `@amd64/@arm64/@heterogeneous` are no longer needed.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml [hunk 3]
```diff
@@ -246,6 +246,45 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: rosa-aws-sts-advanced
+- as: aws-rosa-sts-hcp-advanced-stage-critical-f14
+  cron: 35 6 3,17 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      CHANNEL_GROUP: candidate
+      E2E_RUN_TAGS: '@aws-ipi and @rosa and @hypershift-hosted'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml:255]: Shall we add `and @critical` here ?

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml]: Thanks for your remind, updated

**Pull request merged**